src/config.ml

```diff
@@ -1608,11 +1574,13 @@ let a_inline =
   *> take_while1 (function 'a' .. 'z' | '0' .. '9' | '-' -> true | _ -> false)
   <* char '>' <* a_newline
   >>= fun tag ->
-  skip_many (a_whitespace_or_comment *> end_of_line)
-  *> take_till (function '<' -> true | _ -> false)
-  >>= fun x ->
-  return (`Inline (tag, x))
-  <* skip_many end_of_line <* char '<' <* char '/' <* string tag <* char '>'
+  let end_tag = Printf.ksprintf string "</%s>" tag in
+  commit
+  *> many_till
+       ( take_till (function '\r' | '\n' -> true | _ -> false) >>= fun line ->
+         choice [ string "\r\n"; string "\n" ] >>| ( ^ ) line )
```

Comment:
isn't this `a_newline`? or, asked differently what is our take on carriage returns? simply ignore them or do they have a purpose and we need to preserve them?

Reply:
fine to retain the newlines, but I'd say let's ignore the carriage return ;)

Reply:
I did this and now a test fails because the byte offset reported in an error is now off by one. I am questioning the usefulness of the byte offset reporting so I may remove that...

Reply:
What I find most useful is the output of the surroundings where the parse error happened.